lib/l10n/intl_ru.arb

```
@@ -5751,5 +5751,76 @@
+  },
+  "updateTheLabelName": "Обновить название ярлыка",
+  "@updateTheLabelName": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "areYouSureYouWantToDeleteTheLabel": "Вы уверены, что хотите удалить ярлык \"{labelName}\"?",
+  "@areYouSureYouWantToDeleteTheLabel": {
+    "type": "text",
+    "placeholders_order": [
+      "labelName"
```

RuboCop (1.86.0)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)